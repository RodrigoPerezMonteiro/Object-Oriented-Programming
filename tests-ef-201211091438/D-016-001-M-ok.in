6
4
tomate
20
1
0
0
