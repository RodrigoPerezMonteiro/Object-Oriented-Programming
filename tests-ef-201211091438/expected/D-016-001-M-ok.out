Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: Gestão de Alimentos
1 - Visualizar alimentos
2 - Registar alimento simples
3 - Registar alimento preparado
4 - Alterar alimento simples
5 - Descrever alimento preparado
0 - Sair
Escolha uma opção: Identificador do alimento (nome): Calorias por grama a associar ao alimento (inteiro): Gestão de Alimentos
1 - Visualizar alimentos
2 - Registar alimento simples
3 - Registar alimento preparado
4 - Alterar alimento simples
5 - Descrever alimento preparado
0 - Sair
Escolha uma opção: VEGETABLE|alface|1
VEGETABLE|azeite|9
VEGETABLE|batata|4
MEAT|carne de porco|14
FISH|linguado|6
VEGETABLE|tomate|20
Gestão de Alimentos
1 - Visualizar alimentos
2 - Registar alimento simples
3 - Registar alimento preparado
4 - Alterar alimento simples
5 - Descrever alimento preparado
0 - Sair
Escolha uma opção: Gestão de uma Cadeia de Restaurantes
1 - Novo
2 - Abrir
3 - Guardar
4 - Guardar Como...
5 - Gestão de Clientes
6 - Gestão de Alimentos
7 - Gestão de Restaurantes
8 - Consultas
0 - Sair
Escolha uma opção: 
